app/src/main/res/values-pt/strings.xml

```diff
@@ -25,4 +25,15 @@
     <string name="theme_night">Escuro</string>
     <string name="theme_battery">Seguir a economia de bateria</string>
     <string name="theme">Tema</string>
+    <string name="shortcut">Shortcut</string>
+    <string name="icon_picker">Icon Picker</string>
+    <string name="create_or_append_to_an_existing_shortcut_rather_than_apply">Create or append to an existing shortcut rather than applying it</string>
+    <string name="edit_shortcuts">Edit shortcuts</string>
+    <string name="grant_secure_setting">Grant secure setting</string>
+    <string name="append">Append</string>
+    <string name="disable">Disable</string>
+    <string name="enable">Enable</string>
+    <string name="disabled">Disabled</string>
+    <string name="delete_action">Delete Action</string>
+    <string name="cancel">Cancel</string>
```

Comment:
You don't need to add anything in this file. Revert the changes. The translators will add the new translations later.

Reply:
I also understand that.